packages/mobile/src/components/core/Text.tsx

```diff
@@ -99,6 +102,18 @@ export const Text = (props: TextProps) => {
       ? childrenProp.replace('\n', ' ')
       : childrenProp
 
+  if (bulleted) {
```

Comment:
this is probably fine for now, but some thoughts:
- we're trying to move towards harmony (our design system). don't really expect you to convert the upload to harmony right now tho
- would be better if we could not special-case bulleted.. but again since this is not harmony i don't care so much.